resolwe_bio/docker_images/rnaseq/Dockerfile

```diff
@@ -0,0 +1,83 @@
+FROM docker.io/resolwe/base:ubuntu-16.04
+
+LABEL MAINTAINER Resolwe Bioinformatics authors https://github.com/genialis/resolwe-bio
+
+RUN apt-get update && \
+    apt-get -y install --no-install-recommends software-properties-common && \
```

Comment:
Indeed, this is only needed to add packages from PPAs. I don't see any PPAs being used here, so I think it can be removed.